app/experimenter/legacy-ui/rapid/components/experiments/tooltips.tsx

```diff
@@ -0,0 +1,14 @@
+export const SIGNIFICANCE_TIPS = {
```

Comment:
Good point. I can rename the file. It's definitely a constants file and it will be getting much bigger so I figured it should be separate.

What do you think might be a good spot for it? Maybe `rapid/constants/`?